src/Core/Ast/Parser/NikicPhpParser/FileReferenceVisitor.php

```diff
@@ -60,7 +60,14 @@ private function templatesFromDocs(
         if (null === $docComment) {
             return [];
         }
-        $tokens = new TokenIterator($this->lexer->tokenize($docComment->getText()));
+        $docText = $docComment->getText();
+
+        // prevent expensive parsing, when not templates involved.
```

Comment:
Could `@template` have tool-specific prefixes?

Reply:
I asked myself the same question. .. did not find any tests for it

Reply:
I think I will just drop the `@` so it parses any comment which contains the word template. 

Should be good enough

Reply:
`@template` annotation is used primarily by:
 - Psalm: https://psalm.dev/docs/annotating_code/templated_annotations/#template-psalm-template (they can have tool-specific prefix)
 - PHPStan: https://phpstan.org/blog/generics-in-php-using-phpdocs#ide-compatibility (they can have tool-specific prefix)

However, I would posit that the point of tool-specific prefixes is that they should not be used by other tools. Ergo `deptrac` should only look at `@template` or possibly  `@deptrac-template` but should ignore `@phpstan-template` and `@psalm-template`.

Reply:
I did a few tests locally. deptrac does not pick up tool specific phpdocs as of now.
these are ignored before this PR, therefore I reverted to the initial state.